django/forms/widgets.py

```diff
@@ -78,10 +79,24 @@ def render_css(self):
         ] for medium in media])
 
     def absolute_path(self, path, prefix=None):
+        """
+        Return absolute path to static asset.
+
+        Both absolute and relative paths are accepted.
+        An absolute path will be returned unchanged, where an relative
+        path will be transformed into the staticfile storage absolute path.
+
+        :param path: Relative or absolute path to static asset.
+        :type path: str
+        :param prefix: Added only if ``django.contrib.staticfiles`` is not enabled.
```

Comment:
This would need approval by a core contributor, but as it's an undocumented API that probably isn't used it might be acceptable to just remove it and document the removal. If not, use a normal deprecation approach, which would mean marking as deprecated now and removing it I think at 2.0 (because it wasn't deprecated at 1.8 LTS, so it needs to still work with deprecation warning at 1.11 LTS).